src/components/marker-table/index.js

```diff
@@ -107,6 +107,12 @@ class MarkerTree {
             category = name;
             name = data.name;
             break;
+          case 'DiskIO':
+            category = data.type;
+            name = data.filename
+              ? `${data.operation} – ${data.filename}`
+              : data.operation;
```

Comment:
I think it would be useful to see the source if it's present. Like the DiskIO markers triggered by sqlite operations have a source but no filename, and I think this would be useful to see.